client/tests/integration/multisig.rs

```diff
@@ -80,7 +80,8 @@ fn mutlisig() -> Result<()> {
             .map(Register::account),
     )?;
 
-    let call_trigger = ExecuteTrigger::new(multisig_register_trigger_id).with_args(&args);
+    let call_trigger = ExecuteTrigger::new(multisig_register_trigger_id)
+        .with_args(serde_json::to_value(&args).unwrap());
```

Comment:
this looks like a step back

Reply:
I removed implicit conversions on purpose.

Before there was `From<T: Serialize> for JsonValue`, and was able to panic implicitly if for some reason `T` fails to serialize as JSON. Now it is explicit, at least.

I tried to make emphasis on relying on the "standard" way of working with JSON in Rust, i.e. on `serde_json`. 

Actually, this `serde_json::to_value(&args).unwrap()` could be shortcut to just `json!(&args)`. And I would strongly prefer that rather than adding some methods to `JsonValue`, making it unintuitive to use.